.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/Moya/Moya
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+      LANG: en_US.UTF-8
+      BUNDLE_PATH: vendor/bundle
+    macos:
+      xcode: '9.2.0'
+    steps:
+      - checkout
+      - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+      - restore_cache:
+          keys:
+            - v1-dep-{{ .Branch }}-
+            - v1-dep-master-
+            - v1-dep-
+      - run:
+          name: Set Ruby Version
+          command: echo "ruby-2.4" > ~/.ruby-version
+      - run: 
+          name: Fetch CocoaPods Specs
+          command: curl -sS https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash
+      - run:
+          name: Update Homebrew
+          command: brew update
+      - run:
+          name: Install Swiftlint
+          command: brew install swiftlint
+      - run:
+          name: Install Ruby Dependencies
+          command: bundle check || bundle install
+          environment:
+            BUNDLE_JOBS: 4
+            BUNDLE_RETRY: 3
+      - run: 
+          name: Bootstrap Carthage
+          command: scripts/bootstrap-if-needed.sh
+      - save_cache:
+          key: v1-dep-{{ .Branch }}-{{ epoch }}
+          paths:
+            - vendor/bundle
+            - Carthage
+      - run: 
+          name: Test on iOS
+          command: rake test:ios
+      - run: 
+          name: Test on macOS
+          command: rake test:macos
+      - run:
+          name: Test on tvOS
+          command: rake test:macos
```

Comment:
I think you meant `rake test:tvos` here

Reply:
Good catch !

Reply:
He always had eyes like a hawk 😆 #1 reviewer 🥇